### PR TITLE
Nita Eduard Assignment 01

### PR DESCRIPTION
**Pull request by snoobism**

### CODE REVIEW

File: Students/Nita Eduard Marian/FirstAssignment/FirstAssignment.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		System.out.println(nameInverter("   FirstName   LastName   "));
+		System.out.println(nameInverter("Mr. FirstName LastName"));
+		System.out.println(nameInverter("Ms. FirstName LastName"));
+		System.out.println(nameInverter("Mr. Phd. FirstName LastName"));
+		System.out.println(nameInverter("Ms. Phd. FirstName LastName"));
+		System.out.println(nameInverter("Phd. Ms. FirstName LastName")); //Phd. and Ms. switched
+		System.out.println(nameInverter("One Two Three Four Five")); // More than 4 strings
+	} 
+	public static String nameInverter(String inputString)
+	{
```
Comment (AlexBondor) [inline Students/Nita Eduard Marian/FirstAssignment/FirstAssignment.java]: please stick to one type of curly braces

File: Students/Nita Eduard Marian/FirstAssignment/FirstAssignment.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		System.out.println(nameInverter("   SingleWord    "));
+		System.out.println(nameInverter("   FirstName   LastName   "));
+		System.out.println(nameInverter("Mr. FirstName LastName"));
+		System.out.println(nameInverter("Ms. FirstName LastName"));
+		System.out.println(nameInverter("Mr. Phd. FirstName LastName"));
+		System.out.println(nameInverter("Ms. Phd. FirstName LastName"));
+		System.out.println(nameInverter("Phd. Ms. FirstName LastName")); //Phd. and Ms. switched
+		System.out.println(nameInverter("One Two Three Four Five")); // More than 4 strings
+	} 
+	public static String nameInverter(String inputString)
```
Comment (AlexBondor) [inline Students/Nita Eduard Marian/FirstAssignment/FirstAssignment.java]: usually there's a newline between methods

File: Students/Nita Eduard Marian/FirstAssignment/FirstAssignmentTest.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+class FirstAssignmentTest {
+
+	@Test
+	void test() {
+		Assert.assertTrue(!(FirstAssignment.nameInverter("Mr. FirstName   LastName").equals("invalid input")));
+		Assert.assertTrue(!(FirstAssignment.nameInverter("  singleWord   ").equals("invalid input")));
+		Assert.assertTrue(!(FirstAssignment.nameInverter("Mr. Phd. FirstName   LastName").equals("invalid input")));
+		Assert.assertTrue(!(FirstAssignment.nameInverter("").equals("invalid input")));
+		// why is the null test invalid?
+		Assert.assertTrue(!(FirstAssignment.nameInverter(null).equals("invalid input")));
```
Comment (AlexBondor) [inline Students/Nita Eduard Marian/FirstAssignment/FirstAssignmentTest.java]: actually it should throw a `NullPointerException`..  beccause `FirstAssignment.nameInverter(null)` gets evaluated to `null` and then you're trying to call the `equals` method on it.. just like `null.equals(..)`. You're trying to access a resource (the method) on a `null` object

Comment (snoobism) [reply Students/Nita Eduard Marian/FirstAssignment/FirstAssignmentTest.java]: I see, thought `.equals()` could be called on anything. Thanks for clarifying it!

**Pull request merged**